protocol/contracts/farm/facets/SiloFacet/SiloFacet.sol

```diff
@@ -82,25 +82,66 @@ contract SiloFacet is TokenSilo {
      */
 
     function transferDeposit(
+        address sender,
```

Comment:
@BeanstalkFarms/frontend: Requires ABI update and small tweak to the Transfer form.